### ISSUE TITLE
FileApi.listLines: Wrong naming of parameter

### ISSUE THREAD

**Issue by l0wlik34G6** [cdancy/bitbucket-rest#230]
In the FileApi, there is the function listLines, which gives plenty of parameters.
One of the parameters based on the Bitbucket documentation is "at", in your project it's "commitHash".

Unfortunately this is very confusing, as it is not referencing to a specific commit hash, but to a branch e.g.

So calling the API with `?at=refs/heads/master` or `?at=refs/tags/%myTag%` will give a valid result.
Putting a commitHash instead would produce an error message:
````
{
    "errors": [
        {
            "context": null,
            "exceptionName": "com.atlassian.bitbucket.content.NoSuchPathException",
            "message": "The path \"xxx\" does not exist at revision \"9aa913957f1...\""
        }
    ]
}
````

### Expected Behavior
To avoid misunderstanding, the parameter should be named like in Bitbucket documentation as there is at least some documentation there

### Current Behavior
The parameter `commitHash` is named wrongly and should be renamed

### Context
It is just annoying and misleading if you deal with that

### Your Environment
Version of bitbucket-rest: 2.6.3
Version of Bitbucket: 7.4.0

**Comment by cdancy** [comment]
@l0wlik34G6 the parameter names I used are simply the ones coming from the Bitbucket REST API itself. I tried to make things as close to an apples-to-apples comparison so that developers looking over the REST API would feel at home here. I see in the documentation [HERE](https://docs.atlassian.com/bitbucket-server/rest/5.1.0/bitbucket-rest.html#idm45588158357840) that `at` is indeed used. Are you seeing something somewhere else?
